log/handler_glog.go

```diff
@@ -168,8 +168,8 @@ func (h *GlogHandler) WithGroup(name string) slog.Handler {
 	panic("not implemented")
 }
 
-// Log implements Handler.Log, filtering a log record through the global, local
-// and backtrace filters, finally emitting it if either allow it through.
+// Handle implements slog.Handler.Handle, filtering a log record through the global, local
```

Comment:
Please fix this comment:

```
Handle implements slog.Handler, filtering a log ...
```

Reply:
Hi sir, did you mean that I should update whole comments to illustrate the function of `func Handle`?

Reply:
I think he actually asks to change this:
`slog.Handler.Handle`
=>
`slog.Handler`

Reply:
@rjl493456442 Hi sir, I have updated the function comments, thanks for your review~